src/algokit/cli/doctor.py

```diff
@@ -42,23 +43,25 @@ def doctor_command(*, copy_to_clipboard: bool) -> None:
     potential issues."""
     os_type = platform.system()
     is_windows = get_is_windows()
+    container_engine = get_container_engine()
+    docs_url = f"https://{container_engine}.io"
+    compose_minimum_version = get_min_compose_version()
     service_outputs = {
         "timestamp": DoctorResult(ok=True, output=dt.datetime.now(dt.timezone.utc).replace(microsecond=0).isoformat()),
         "AlgoKit": _get_algokit_version_output(),
         "AlgoKit Python": DoctorResult(ok=True, output=f"{sys.version} (location: {sys.prefix})"),
         "OS": DoctorResult(ok=True, output=platform.platform()),
-        "docker": check_dependency(
-            ["docker", "--version"],
-            missing_help=[
-                "Docker required to run `algokit localnet` command; install via https://docs.docker.com/get-docker/"
-            ],
+        container_engine: check_dependency(
+            [container_engine, "--version"],
+            missing_help=[f"`{container_engine}` required to run `algokit localnet` command; install via {docs_url}"],
         ),
-        "docker compose": check_dependency(
-            DOCKER_COMPOSE_VERSION_COMMAND,
-            minimum_version=DOCKER_COMPOSE_MINIMUM_VERSION,
+        f"{container_engine} compose": check_dependency(
```

Comment:
Does the podman compose equivalent follow the same versioning as docker?

Reply:
@neilcampbell no but the get_min_compose_version is dynamically resolving the value